### ISSUE TITLE
[question] How would you restrict access to the server instance?

### ISSUE THREAD

**Issue by o-alquimista** [rfc1036/udptunnel#10]
udp2raw uses a password.

**Comment by rfc1036** [comment]
Yes: it would be a good idea to modify the programs to optionally send/receive a fixed length secret at the beginning of the connection to have low security access control.
